File: scraper/article.py
import json


class Article:

    def __init__(self, url, headline, weight, webpage_text="", tickers="", paragraphs=[]):
        self.webpage_text = webpage_text
        self.tickers = tickers
        self.url = url
        self.headline = headline
        self.paragraphs = paragraphs
        self.weight = weight

    def toJson(self):
        return json.dumps({
            "text": self.webpage_text,
            "tickers": self.tickers,
            "url": self.url,
            "headline": self.headline,
            "paragraphs": self.paragraphs,
            "weight": self.weight
        })

    def toObj(self):
        return {
            "text": self.webpage_text,
            "tickers": self.tickers,
            "url": self.url,
            "headline": self.headline,
            "paragraphs": self.paragraphs,
            "weight": self.weight
        }

File: NLP/java-project/src/main/java/com/hackathon/NLP/Models/Article.java
package com.hackathon.NLP.Models;

import java.util.ArrayList;

public class Article {
    public String webpage_text;
    public ArrayList<String> tickers = new ArrayList<>();
    public String url;
    public String headline;
    public ArrayList<Paragraph> paragraphs = new ArrayList<>();
    public int weight;
}
